Samples/HttpTrigger2/function.json

```diff
@@ -0,0 +1,17 @@
+{
+  "scriptFile": "main.py",
+  "disabled": false,
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "res"

```

Comment:
Output binding tied to the return value must use `name : $return`

Reply:
Makes sense.